### PR TITLE
 "Hello Microverse" project

### PR DESCRIPTION
**Pull request by Yonashenok**
First Microverse project

1. I make sure to set up a linter for HTML and CSS and fixed linter error
2. I have made modification on document (README).
3.  Also added .gitignore file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (joel-kalema) [inline index.html:2]: #### Good job working on setting up a 'Hello Microverse' project 👍. However, you need to fix some parts :

- [ ] Currently, the project doesn't have a linter installed as it is [required](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=Set%20up%20a%20linter%20for%20HTML%20and%20CSS%20and%20make%20sure%20that%20any%20files%20that%20are%20not%20relevant%20are%20ignored%20by%20git..), Kindly visit this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) and follow the steps to install linters 👍  
- [ ] Also currently your project doesn't have a descriptive README as it is [required](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=Add%20descriptive%20README%20file%20to%20your%20project%20%2D%20please%20use%20this%20template%20but%20remember%20to%20customize%20it%20to%20your%20project.), Kindly us this [README temple](https://github.com/microverseinc/readme-template) and customize it to make it more descriptive 👍
- [ ]  Also, make sure your pull request is more descriptive, you may add more information about what you have done in this project 👍

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Yonashenok 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (V-Blaze) [inline README.md:1]: - [OPTIONAL] You did a great job implementing the `README` file suggested by the previous reviewer and also customizing the content. :+1: :100: , I think you can make it more professional by removing the broken logo and also removing the comment `Microverse README Template`.
- Screenshot for reference:
Screenshot from 2023-04-25 17-39-56

**Review by V-Blaze — APPROVED** [review]
Hi @Yonashenok  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved-accepted


## Highlights

- You implemented the linters correctly 💯
-  Pull request description is detailed👍
- You used correct readme template🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (Yonashenok) [reply README.md:1]: Thanks a lot, will fix it soon

**Pull request merged**